datalad/cmdline/helpers.py

```diff
@@ -41,8 +41,8 @@ def __call__(self, parser, namespace, values, option_string=None):
                 if not os.path.exists(manfile):
                     raise IOError("manfile is not found")
                 with gzip.open(manfile) as f:
-                    man_th = [line for line in f if line.startswith(".TH")][0]
-                man_version = man_th.split(' ')[5].strip(" '\"\t\n")
+                    man_th = [line for line in f if line.startswith(b".TH")][0]
+                man_version = man_th.split(b' ')[5].strip(b" '\"\t\n").decode('utf-8')
```

Comment:
hard to get there without much of mocking, didn't have time etc... so will leave it uncovered ;)